src/instructlab/process/process.py

```diff
@@ -379,16 +379,9 @@ def list_processes():
 
     list_of_processes = []
 
-    processes_to_remove = []
     for local_uuid, process in process_registry.processes.items():
-        # assume all processes are running and not ready for removal unless status indicates otherwise
-        if not all_processes_running(process.pids):
-            # if all of our child or parent processes are not running, we should either a. remove this process, or b. mark it ready for removal after this list
-            if process.completed:
-                # if this has been marked as done remove it after listing once
-                # but, we cannot remove while looping as that will cause errors.
-                processes_to_remove.append(local_uuid)
-            # if not, list it, but mark it as ready for removal
+        if state.lower() != "all" and process.status.lower() != state.lower():
```

Comment:
The code is sprinkled with "All" string. It's probably ok for cli/ module change (though I'd rather have a constant defined for it too), but this is now leaking into unrelated module. Consider adding a default value for `state` argument (`state: str = ""`?) and use its falsy-ness as indication that state filter shouldn't be applied.